test/unit/helpers/bignum.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+var chai = require('chai');
+var expect = require('chai').expect;
+var bignum = require('../../../helpers/bignum');
+
+describe('bignum', function () {
+
+	var SEED = '782910138827292261791972728324982';
+	var BUFFER_SEED = Buffer.from('Jpm3GDWJ6Efns5p1Q3Y=', 'base64');
+	var BUFFER_SEED_SIZE_TWO = Buffer.from('mSYYt4k1R+iz53WadkM=', 'base64');
+	var BUFFER_SEED_MPINT = Buffer.from('AAAADiaZtxg1iehH57OadUN2', 'base64');
+
+	describe('toBuffer', function () {
+
+		var testbignum;
+
+		function standardExpect (buffer, shouldEqual) {
+			expect(buffer.equals(shouldEqual)).to.be.true;
+		}
+
+		before(function () {
```

Comment:
Let's make this a `beforeEach` to guarantee atomicity. (Otherwise you may as well assign it when declaring `testbignum`.)

Reply:
complete.